www/experiments/findings.inc

```diff
@@ -23,86 +24,137 @@ if ($experiment) {
-                        echo '<td id="TTFB" valign="middle" class="' . ( $thisDiff >= 0 ? "good" : "poor" ) . '">' . abs($thisDiff) . $units . $change . $compare . '</td>';
                     }
-                }
 
                     // additional non-metric comparisons
-                if ($blockingReqsDiff !== 0) {
-                    $changeTerms = array("Fewer","More");
-                    $units = "";
-                    $change = '<span class="metric_change"><em>' . ( $blockingReqsDiff >= 0 ? $changeTerms[0] : $changeTerms[1] ) . '</em></span>';
-                    $compare = '<span class="metric_experiment"><span>Experiment:</span><span>' . count($experimentBlockingReqs) .  $units . '</span></span>';
-                    $compare .= '<span class="metric_control"><span>Control:</span><span>' . count($controlBlockingReqs) .  $units . '</span></span>';
-
-                    echo '<td id="TTFB" valign="middle" class="' . ( $blockingReqsDiff >= 0 ? "good" : "poor" ) . '">' . abs($blockingReqsDiff) . $units . $change . $compare . '</td>';
-                }
-                ?>
-            </tr>
-        </tbody>
-    </table>
-</div>
-
-
-<?php } ?>
\ No newline at end of file
+                    if ($blockingReqsDiff !== 0) {
+                        $changeTerms = array("Fewer", "More");
+                        $units = "";
+                        $change = '<span class="metric_change"><em>' . ($blockingReqsDiff >= 0 ? $changeTerms[0] : $changeTerms[1]) . '</em></span>';
+                        $compare = '<span class="metric_experiment"><span>Experiment:</span><span>' . count($experimentBlockingReqs) .  $units . '</span></span>';
+                        $compare .= '<span class="metric_control"><span>Control:</span><span>' . count($controlBlockingReqs) .  $units . '</span></span>';
+
+                        echo '<td id="TTFB" valign="middle" class="' . ($blockingReqsDiff >= 0 ? "good" : "poor") . '">' . abs($blockingReqsDiff) . $units . $change . $compare . '</td>';
+                    }
+                    ?>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+
+
+<?php }
+
+if ($ttfbDiffNotable) {
+    if ($removeTTFBfromDiffs) {
+        echo "<p class=\"ttfb_experiment_warning\"><strong>Note:</strong> This comparison view is omitting TTFB to adjust for network variance. <a href=\"" . $experimentResultsHref . "\">View this comparison with TTFB differences visible.</a>";
+    } else {
+        echo "<p class=\"ttfb_experiment_warning\"><strong>Note:</strong> We noticed a " . $ttfbDiffClean . "s difference in response time between the experiment and control runs, which are usually unrelated to the experiment and can might make other metric changes seem more significant than they really are. <a href=\"" . $experimentResultsHref . "&ignoreTTFB\">View this comparison with TTFB differences removed</a>";
```

Comment:
I'd say single quotes here will be easier to read, since you're concatenating the php vars and not inlining them
```
echo '<p class="ttfb...
```